docs/proposals/scheduling/20220701-schedule-gang.md

```diff
@@ -146,9 +146,9 @@ Pod should use `pod-group.scheduling.sigs.k8s.io` in label to associate with `Po
 
 Also, we introduce some optional definitions as below:
 ```yaml
-gang.scheduling.koordinator.sh/total-number
-gang.scheduling.koordinator.sh/gang-mode        
-gang.scheduling.koordinator.sh/groups
+coescheduling.scheduling.koordinator.sh/total-number
```

Comment:
Does our custom protocol still need to be modified to the `coscheduling` prefix?